test/channel/base_test.rb

```diff
@@ -92,6 +111,23 @@ def rm_rf
     assert_nil @channel.last_action
   end
 
+  test "try calling a public method defined on Base" do
+    @channel.perform_action 'action' => :kick
+    assert_nil @channel.last_action
```

Comment:
So `kick` sets `@last_action`, but here we're asserting that last action is nil - is this supposed to mean that the action isn't allowed? I really don't understand it :smile: